docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -157,3 +157,47 @@ kubectl get secrets -o json |
         {auths: .}
     '
 ----
+
+== Manually Update Task Bundles
+
+Usually, Konflux users rely on link:https://docs.renovatebot.com/[renovate] to update
+the various Task bundle references in the build Pipelines. However, it is also possible
+to update these references manually if needed. The script below provides a working
```

Comment:
Can you provide an overview of what this is doing? While what you are describing is manual, it is a self-triggered automated approach. Describing an _actual_ manual approach and then following up with this script might be helpful as well?

Reply:
Done.